SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -660,3 +660,35 @@ Worker_ComponentId USpatialClassInfoManager::ComputeActorInterestComponentId(con
 	}
 	return SpatialConstants::INVALID_COMPONENT_ID;
 }
+
+bool USpatialClassInfoManager::IsInterestBucketComponentId(const Worker_ComponentId ComponentId) const
+{
+	if (ComponentId == SpatialConstants::SERVER_ONLY_ALWAYS_RELEVANT_COMPONENT_ID)
```

Comment:
could also make this

```suggestion
	return ComponentId == SpatialConstants::SERVER_ONLY_ALWAYS_RELEVANT_COMPONENT_ID || 
	ComponentId == SpatialConstants::ALWAYS_RELEVANT_COMPONENT_ID || 
	IsNetCullDistanceComponent(ComponentId)
	
```